www/templates/control/main-control.html

```diff
@@ -5,7 +5,13 @@
       <div ng-click="logOut()" id ="switch-user" class="control-icon-button"><i class="icon ion-log-out"></i></div>
     </div>
     <enketo-demographics-button></enketo-demographics-button>
-    <div class="control-list-item">
+    <!-- each list item is very similar:
+      contains a text and a button icon or switch (for the most part) 
+      moving to react -- a text component and an icon button component that live in a list item comoponent? -->
+    <!-- <div class="control-list-item">
+      <control-list-item text="View Privacy" icon-name = "eye"></control-list-item>
+    </div> -->
+      <div class="control-list-item">
       <div class="control-list-text" ng-i18next>{{'control.view-privacy'}}</div>
```

Comment:
This change is inadvertent, right?

Reply:
Yes, part of the same thing you noted in the `general-settings.js`